.travis.yml

```diff
@@ -98,6 +98,8 @@ install:
 script: |
   set -e
   if [[ $JOB = docs ]]; then
+    # install docs dependencies
+    pip install '.[docs]'
```

Comment:
thanks that makes sense. So this means that we should only do `install '.[docs]'\
and put all the other extra requirements into the setup.py as well?
I moved this line up for now.